.pre-commit-config.yaml

```diff
@@ -22,7 +22,8 @@ repos:
       - id: check-case-conflict # Avoid case sensitivity in file names.
       - id: debug-statements # Watch for lingering debugger calls.
       - id: end-of-file-fixer # Ensure there's a newline at EOF.
-      - id: mixed-line-ending # Only newlines, no line-feeds.
+      - id: mixed-line-ending # Use Unix line-endings to avoid big no-op CSV diffs.
+        args: ["--fix=lf"]
```

Comment:
This is another way of ensuring that all of the text files have standard line endings. Probably don't need both the git and the pre-commit hook versions but they also don't seem to conflict with each other, and the pre-commit hook will be forcibly applied by pre-commit-ci as a backup.